### PR TITLE
Done with multi counter task.

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Two counters in separate threads counting to different max value, when one of then finishes first the other stops with it.

### CODE REVIEW

File: counter/src/main/java/com/clouway/counter/Counter.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    /**
+     * Increase count after 1000 milliseconds.
+     *
+     * @return Last counted number till reaching max or interrupted.
+     */
+    private String increment() {
+        while (count < max) {
+            try {
+                Thread.sleep(1000);
+                count++;
```
Comment (EmilGeorgiev) [inline counter/src/main/java/com/clouway/counter/Counter.java]:
```text
На всяко отброяване да се извежда стойността на брояча.
```

File: counter/src/main/java/com/clouway/counter/ThreadMonitor.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                thread.interrupt();
+                try {
+                    thread.join();
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+                return thread.getName() + " interrupted!";
+            }
+        }
+        return null;
```
Comment (EmilGeorgiev) [inline counter/src/main/java/com/clouway/counter/ThreadMonitor.java]: it is not good methods in java to return null.

Comment (AlexanderVladimirov9090) [inline counter/src/main/java/com/clouway/counter/ThreadMonitor.java]: Know just forgot to change it. will be fixed

**Comment by AlexanderVladimirov9090** [comment]
Done with the fix.

File: counter/src/main/java/com/clouway/counter/ThreadMonitor.java (hunk 2 above)
Comment (AlexanderVladimirov9090) [inline counter/src/main/java/com/clouway/counter/ThreadMonitor.java]: and this is done.

**Pull request closed without merging**